test/packages/benchmarks/system_benchmark/_dev/benchmark/system/20mb-logs-benchmark.yml

```diff
@@ -0,0 +1,12 @@
+---
```

Comment:
Why are the benchmarks not already in the `_dev` top level directory? Are these excluded during the build process?